satellite/analytics/service.go

```diff
@@ -101,8 +128,71 @@ func (service *Service) enqueueMessage(message segment.Message) {
 	}
 }
 
+func (service *Service) createHubSpotUser(ctx context.Context, data PayloadUser) (err error) {
+	if !service.config.Enabled {
+		return nil
+	}
+	defer mon.Task()(&ctx)(&err)
+	payloadBytes, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+
+	body := bytes.NewReader(payloadBytes)
+
+	url := "https://api.hubapi.com/crm/v3/objects/contacts?hapikey=" + url.QueryEscape(service.config.HubSpotAPIKey)
```

Comment:
If the hubapi is hardcoded how does do the tests work?

Reply:
There is an `Enabled` flag which is false for the tests, so they end up skipping this part. Additionally, the analytics functions like `TrackCreateUser` don't return errors, so their success/failure is independent of important production code (if they fail, we can still deliver our service, even if we don't get the analytics). Based on my conversations with Prerna, she has tested that the analytics are being sent correctly to Hubspot in storj-sim with analytics enabled. Given all this, I don't think it is necessary to create a mock server just for tests on these analytics endpoints.
What do you think?